rs/http_endpoints/public/src/lib.rs

```diff
@@ -305,14 +305,13 @@ pub fn start_server(
     let listen_addr = config.listen_addr;
     info!(log, "Starting HTTP server...");
 
-    let _enter = rt_handle.enter();
     // TODO(OR4-60): temporarily listen on [::] so that we accept both IPv4 and
     // IPv6 connections. This requires net.ipv6.bindv6only = 0. Revert this once
     // we have rolled out IPv6 in prometheus and ic_p8s_service_discovery.
     let mut addr = "[::]:8080".parse::<SocketAddr>().unwrap();
     addr.set_port(listen_addr.port());
-    let tcp_listener = start_tcp_listener(addr);
-
+    let tcp_listener = start_tcp_listener(addr, &rt_handle);
+    let _enter = rt_handle.enter();
```

Comment:
Same

Reply:
Yes, because some of the tower layer require the runtime even on construction